bot/exts/moderation/infraction/_scheduler.py

```diff
@@ -1,6 +1,7 @@
 import textwrap
 import typing as t
 from abc import abstractmethod
+from collections.abc import Callable
```

Comment:
Out of curiosity, why use `collections.abc` rather than `typing` which is already imported

Reply:
Because of [PEP 585](https://peps.python.org/pep-0585/), `typing.Callable` is deprecated.

We could probably do a PR to switch over everything at some point, but there's no real rush. From the PEP:
> The deprecated functionality will be removed from the typing module in the first Python version released 5 years after the release of Python 3.9.0.